airbyte-ci/connectors/metadata_service/lib/tests/fixtures/metadata_validate/valid/metadata_connector_base_image.yaml

```diff
@@ -0,0 +1,28 @@
+data:
+  allowedHosts:
+    hosts:
+      - "*.googleapis.com"
+  connectorBaseImage: airbyte/airbyte-python-connectors-base:0.1.0
```

Comment:
@alafanechere What are your thoughts around placing these types of fields under a `build` sections

```
data:
    build:
        connectorBaseImage: airbyte/airbyte-python-connectors-base:0.1.0
```

Reply:
@bnchrch I made the change. I find using the field name `baseImage` more concise.